examples/employee-app/src/main/resources/employees.json

```diff
@@ -0,0 +1,40 @@
+[{"firstName":"Hugh","lastName":"Jast","email":"Hugh.Jast@example.com","phone":"730-715-4446","birthDate":"1970-11-28","title":"National Data Strategist","department":"Mobility"},
```

Comment:
This file could be formatted a little bit I think.

Reply:
Done